tests/rules/avoid-duplicate-actions-in-reducer.test.ts

```diff
@@ -1,176 +1,183 @@
+import type { InvalidTestCase } from '@typescript-eslint/experimental-utils/dist/ts-eslint'
 import { stripIndents } from 'common-tags'
 import path from 'path'
+import { test } from 'uvu'
+import type { MessageIds } from '../../src/rules/store/avoid-duplicate-actions-in-reducer'
 import rule, {
   avoidDuplicateActionsInReducer,
   avoidDuplicateActionsInReducerSuggest,
 } from '../../src/rules/store/avoid-duplicate-actions-in-reducer'
 import { ruleTester } from '../utils'
 
-ruleTester().run(path.parse(__filename).name, rule, {
-  valid: [
-    `
+const valid = [
+  `
+  export const reducer = createReducer(
+    {},
+    on(abc, state => state),
+    on(def, state => state),
+    on(ghi, state => state),
+  )`,
+  `
+  export const reducer = createReducer(
+    {},
+    on(abc, state => state),
+    on(def, state => state),
+    on(ghi, state => state),
+  )
+
+  export const reducerTwo = createReducer(
+    {},
+    on(abc, state => state),
+    on(def, state => state),
+    on(ghi, state => state),
+  )`,
+  // does not crash when no arguments present
+  `
+  export const reducer = createReducer(
+    {},
+    on(),
+  )`,
+]
```

Comment:
I'd add this where possible:

```suggestion
] as const
```

Reply:
Where would this be needed?